ACT compiler warning on Cray compiler

ACT/src/gromacs/utility/real.h:164:27: warning: identifier '_real' preceded by whitespace in a literal operator declaration is deprecated [-Wdeprecated-literal-operator]
  164 | constexpr real operator"" _real(long double x) { return real(x); }
      |                ~~~~~~~~~~~^~~~~
      |                operator""_real

--- a/src/gromacs/utility/real.h
+++ b/src/gromacs/utility/real.h
@@ -161,6 +161,6 @@ typedef float           real;
  *
  * See https://en.cppreference.com/w/cpp/language/user_literal for details on this lanuage feature.
  */
-constexpr real operator"" _real(long double x) { return real(x); }
+constexpr real operator""_real(long double x) { return real(x); }
 
 #endif

